docs/source/migration.mdx

```diff
@@ -442,11 +441,15 @@ See [GraphQL Playground](#graphql-playground).
 
 In Apollo Server 2, `apollo-server` and framework integration packages such as `apollo-server-express` import many symbols from third-party packages and re-export them. This effectively ties the API of Apollo Server to a specific version of those third-party packages and makes it challenging to upgrade to a newer version or for you to upgrade those packages yourself.
 
-In Apollo Server 3, most of these "re-exports" are removed. If you want to use these exports, you should import them directly from their originating package
+In Apollo Server 3, most of these "re-exports" are removed. If you want to use these exports, you should import them directly from their originating package.
 
 ### Exports from `graphql-tools`
 
-Apollo Server 2 exported every symbol exported by [`graphql-tools`](https://www.graphql-tools.com/) v4. If you're importing any of the following symbols from an Apollo Server package, you should instead run `npm install graphql-tools@4.x` and import the symbol from `graphql-tools` instead. Alternatively, read the [GraphQL Tools docs](https://www.graphql-tools.com/docs/introduction) and find out which `@graphql-tools/subpackage` the symbol is exported from in more modern versions of GraphQL Tools.
+Apollo Server 2 exports every symbol exported by [`graphql-tools`](https://www.graphql-tools.com/) v4. If you're importing any of the following symbols from an Apollo Server package, you should instead run `npm install graphql-tools@4.x` and import the symbol from `graphql-tools`.
+
+> Alternatively, read the [GraphQL Tools docs](https://www.graphql-tools.com/docs/introduction) and find out which `@graphql-tools/subpackage` the symbol is exported from in more modern versions of GraphQL Tools.
+
+<ExpansionPanel title="Click to view symbols">
```

Comment:
While the lack of an expansion panel does make the document longer, it also means you can immediately search for the name of the symbol involved in the error you're seeing upon upgrade and get a result. With the expansion panel you search for the symbol and get no results. I think it's better without. Maybe you have some split-the-difference way out of this though?

Reply:
Don't think I do but I respect the use case. I'll remove the expander